### PR TITLE
Remove deprecated code! :)

### PR DESCRIPTION
**Pull request by lcapelluto**
### Summary
Remove code that has been deprecated 3 months:
  - CmdDef
  - buffer
  - methods in InstructionScheduleMap
  - warning about method behavior change (GHz to Hz in pulse defaults)

### CODE REVIEW

**Comment by taalexander** [comment]
Seems like some imports were missing, good after that.

**Comment by nkanazawa1989** [comment]
is there any plan for deprecation of PVs?

**Comment by lcapelluto** [comment]
> is there any plan for deprecation of PVs?

@nkanazawa1989 yes, it's already deprecated, but it's only been 2 months, so we can't remove it yet.

File: releasenotes/notes/remove-deprecated-code-e48bbdd7cb0d61db.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+upgrade:
+  - |
+    ``CmdDef`` has been deprecated since Terra 0.11.0 and has been removed.
+    Please continue to use :py:class:`~qiskit.pulse.InstructionScheduleMap`
+    instead.
+  - |
+    The methods ``cmds`` and ``cmd_qubits`` in
+    :py:class:`~qiskit.pulse.InstructionScheduleMap` have been deprecated
+    since Terra 0.11.0 and have been removed. Please use ``instructions``
+    and ``qubits_with_instruction`` instead.
```
Comment (mtreinish) [inline releasenotes/notes/remove-deprecated-code-e48bbdd7cb0d61db.yaml]: You could use method links for each for both of these, but not a blocker

**Pull request merged**